app/controllers/main/routes.py

```diff
@@ -113,3 +114,9 @@ def RemoveRSVP():
 
     flash("Successfully unregistered for event!", "success")
     return redirect(url_for("admin.editEvent", eventId=eventId, program=program))
+@main_bp.route('/contributors',methods = ['GET'])
+def contributors():
+    contribs = app.config['contributors']
+    return render_template("/Contributors.html",
+           cfg = contribs
```

Comment:
You don't need to pass this into the page. The application config is available to templates https://stackoverflow.com/questions/7104198/flask-accessing-the-config-variable-in-the-template